src/Input/InputLabel.js

```diff
@@ -12,28 +12,34 @@ export const styleSheet = createStyleSheet('MuiInputLabel', (theme) => {
   return {
     root: {
       transformOrigin: 'top left',
+      fontSmoothing: 'antialiased',
```

Comment:
Ohh, that's a tough topic! I think that it far out of the scope of this PR.

1. Is antialiasing better than a sub-pixel smoothing? (I think it's)
2. Do we want to apply it on all the components/the whole page? (I don't think that whole page is a good call for isolation)
3. Who do we apply it to all the components? (I guess that we could create a theme object spread to all the `root` element of our component)